code/part-two/README.md

```diff
@@ -240,22 +240,22 @@ Let's answer these questions for _Cryptomoji_.
 
 ### Encoding Data
 
-For simplicity and familiarity we are going to encode both payloads and state
+For simplicity and familiarity, we are going to encode both payloads and state
 data as JSON. To be clear, JSON would _not_ be a great choice in production. It
-is not space efficient, and worse, it is not _deterministic_. Determinism is
-very important when writing state to the blockchain. Many many nodes will be
-attempting to the write the same state. If that state is even slightly
-different (like say, the keys are in a different order), the validators will
+is not efficient with space; worse, it is not _deterministic_. Determinism
```

Comment:
I'd like to drop this change. "Space efficient" is a technical term with a specific meaning.

Reply:
OK - I'll restore the original version. I changed it so I wouldn't have to research whether "space efficient" should be hyphenated, but the phrase is clear enough (even if the punctuation rules aren't).

Reply:
Seems like both "space efficient" and "space-efficient" are acceptable. I have no preference.